src/desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -65,10 +66,22 @@ module.exports = class ArtworkAuctionView extends Backbone.View
     $target = $(e.currentTarget)
     $target.attr 'data-state', 'loading'
 
-    acquire AUCTION.artwork_id
-      .catch ->
-        $target.attr 'data-state', 'error'
-        location.reload()
+    # Show the new buy now flow if you have the lab feature enabled
+    if CURRENT_USER?.hasLabFeature('New Buy Now Flow')
+      createOrder
+        artworkId: AUCTION.artwork_id
+        quantity: 1
+        user: CURRENT_USER
+      .then (data) ->
```

Comment:
Might this need some error handling /`catch`? Also, with the `?` nil soaking, `order` might wind up being `null` (if there's something unexpected/wrong with the MP response), which would then crash in the next line, on `order.id`.

Reply:
Yeah @mzikherman we are going to follow-up with error handling since that is also new! For now this is okay since it's just for testing purposes.